app/src/main/res/xml/preferences.xml

```diff
@@ -59,11 +64,12 @@
             android:icon="@drawable/ic_fullscreen"
             android:title="@string/fullscreen"
             android:summary="@string/fullscreen_def"/>
-        <Preference
-            android:key="nfc_tags"
-            android:icon="@drawable/ic_plus"
-            android:title="@string/nfc_title_settings"
-            android:summary="@string/nfc_summary" />
+        <ListPreference
```

Comment:
I wonder if we should provide a description about what actually gets changed when we modify the app language.  Users probably want to know that it not only impacts the UI but also the entity names that we send out.

Reply:
Yep, it is probably more useful than current setting value.